appveyor.yml

```diff
@@ -0,0 +1,335 @@
+version: 5.1.0.{build}
+image: Visual Studio 2017
+configuration:
+    - Debug
+    - Release
+environment:
+    global:
+        targetFx: net461
+    matrix:
+        - db: Firebird
+        - db: MySQL
+        - db: Odbc
+        - db: PostgreSQL
+        - db: SQLite
+        - db: SqlServerCe
+        - db: SqlServer2008
+        - db: SqlServer2012
+
+matrix:
+    exclude:
```

Comment:
Debug configuration is tested with all databases, release is done only with SQL 2012.